core/src/cf/mutations.rs

```diff
@@ -21,7 +22,7 @@ pub enum TableMutation {
 	Def(DefineTableStatement),
 	#[revision(start = 2)]
 	// Includes the previous value that may be None
-	SetPrevious(Thing, Value, Value),
+	SetWithDiff(Thing, Value, Vec<Operation>),
```

Comment:
What do `Vec<Operation>` represent here? Could you expand the comment so the meaning of this new Mutation is more explicit?